docusaurus/docs/dev-docs/configurations/database.md

```diff
@@ -562,4 +562,4 @@ In addition to `client` values of '[postgres](https://www.npmjs.com/package/pg)'
 
 :::note
```

Comment:
Not really related to this PR but I just noticed that this whole block can be deleted for v5, as `mysql2` is now the only supported mysql driver and we just call it `mysql`

Reply:
Thank you very much for reporting, Ben! I [removed it](https://github.com/strapi/documentation/commit/ddded0625fa6780e37db7cc8b9b281d215b3a584) directly in the `next` branch (main branch for v5 docs).
I double-checked and there should be no other mentions to mysql2 other than in the [mysql2 breaking change](https://docs-next.strapi.io/dev-docs/migration/v4-to-v5/breaking-changes/only-mysql2-package-for-mysql).